### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by enis-memic**
In this project we did the following:
- Added necessary html elements to the index.html file
- Edited style.css file for the added elements
- Added javascript linters
- Added script.js file with necessary functions
- Edited README.md file, with adding another author to the project

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (gemmen29) [inline .eslintrc.json:1]: - [ ] Kindly update the `linters.yml` file inside `.github/workflow` with the linters in [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) repo in order to run the `eslint` linters as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=For%20this%20project%2C%20you%20should%20add%20the%20JavaScript%20%2B%20HTML%20%2B%20CSS%20linter.). Now the eslint linters check are not running.
image

File: style.css [hunk 2]
```diff
@@ -514,3 +513,59 @@ li {
     width: 385px;
   }
 }
+
```
Comment (gemmen29) [inline style.css:516]: - [ ] Kindly update the color of the close button of the mobile menu to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -15,19 +15,39 @@
   </head>
   <body>
     <header class="header-section">
```
Comment (gemmen29) [inline index.html:17]: - [ ] Kindly make the menu links appear in the desktop version to match the Figma design.

| Your design |
| --- | 
| image | 

| Figma design |
| --- | 
image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @enis-memic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**